bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/team/BedWarsTeam.java

```diff
@@ -219,6 +219,16 @@ public void sendDefaultInventory(Player p, boolean clean) {
                         i.setAmount(Integer.parseInt(parm[2]));
                     }
                     ItemMeta im = i.getItemMeta();
+                    if (!getSwordsEnchantments().isEmpty()) {
+                        if (im == null) continue;
+                        if (nms.isSword(i)) {
+                            for (TeamEnchant e : getSwordsEnchantments()) {
+                                im.addEnchant(e.getEnchantment(), e.getAmplifier(), true);
+                            }
+                            i.setItemMeta(im);
+                        }
+                        p.updateInventory();
+                    }
```

Comment:
This wont work. You are in the sendDefaultInventory function which is only triggered when respawning or at the first spawn

Reply:
> This wont work. You are in the sendDefaultInventory function which is only triggered when respawning or at the first spawn

I tested. it works

Reply:
I can guarantee you this doesnt work

Reply:
sendDefaultInventory is only called in the firstSpawn function and within the respawn member function